extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/PanacheResourceProcessor.java

```diff
@@ -191,7 +211,7 @@ private void lookupNamedQueries(CombinedIndexBuildItem index, DotName name, Set<
         }
 
         // climb up the hierarchy of types
-        if (!classInfo.superClassType().name().equals(DOTNAME_OBJECT)) {
+        if (!classInfo.superClassType().name().equals(io.quarkus.panache.common.deployment.JandexUtil.DOTNAME_OBJECT)) {
```

Comment:
static import of `DOTNAME_OBJECT ` would be better instead of this very long line.

or we could import `JandexUtil` at the very least since it seems to be used everywhere.